images/win/scripts/SoftwareReport/SoftwareReport.Databases.psm1

```diff
@@ -9,6 +9,7 @@ function Get-PostgreSQLMarkdown
         Version = $pgVersion
         UserName = $env:PGUSER
         Password = $env:PGPASSWORD
+        EnvironmentVariables = @{PGBIN=$env:PGROOT; PGDATA=$env:PGDATA; PGROOT=$env:PGBIN}

```

Comment:
Incorrect output in a markdown file: `System.Collections.Hashtable`

Reply:
I have fixed this.